packages/drivers/routerlicious-driver/src/errorUtils.ts

```diff
@@ -124,8 +125,43 @@ export function throwR11sNetworkError(
 export function errorObjectFromSocketError(
 	socketError: IR11sSocketError,
 	handler: string,
+	additionalProps?: DriverErrorTelemetryProps,
 ): R11sError {
 	// pre-0.58 error message prefix: R11sSocketError
 	const message = `R11s socket error (${handler}): ${socketError.message}`;
-	return createR11sNetworkError(message, socketError.code, socketError.retryAfterMs);
+	return createR11sNetworkError(
+		message,
+		socketError.code,
+		socketError.retryAfterMs,
+		additionalProps,
+	);
+}
+
+/** Simulate the pathname for socket connection */
+export const socketIoPath = "socket.io";
```

Comment:
just curious, why this is declared as variable while other path segments are hardcoded ?

Reply:
There isn't a way to get the request URL relating to a socket connection, so I chose to use this const to better detail where failures are coming from (otherwise it would just be a hostname).